includes/core-changes/aspnetcore/3.0/ancmv1-hosting-bundle-removal.md

```diff
@@ -0,0 +1,42 @@
+### AspNetCoreModule V1 removed from Windows Hosting Bundle
+
+The Windows Hosting Bundle won't contain AspNetCoreModule (ANCM) V1 in the ASP.NET Core 3.0 release.
+
+ANCM V2 is backwards compatible with ANCM OutOfProcess and is recommended for use with ASP.NET Core 3.0 apps.
+
+For discussion, see https://github.com/aspnet/AspNetCore/issues/7095.
+
+#### Version introduced
+
+3.0
+
+#### Old behavior
```

Comment:
In a case like this, information is largely being duplicated. It's OK to simply include all of the information in the introductory paragraph (no old and new behavior) along with recommended action, or to have a more succinct intro with a Change description section and a recommended action section.